core-bundle/src/Routing/RouteProvider.php

```diff
@@ -486,11 +487,12 @@ private function findPages(array $candidates): array
 
         if (!empty($aliases)) {
             $conditions[] = 'tl_page.alias IN ('.implode(',', $aliases).')';
+            $options['order'] = sprintf('FIELD(alias, %s)', implode(',', $aliases));
```

Comment:
is this SQL injection safe (because the aliases are taken from the URL)?

Reply:
I have simplified the method in b889f36dc3a610750b68587fdfdfe5d66f7c327d so it works with IDs and aliases and does not need `FIELD()` or `FIND_IN_SET()` at all. Please re-check.